### ISSUE TITLE
Fraidy cat did not render "&#8217;"

### ISSUE THREAD

**Issue by JasonSwindle** [kickscondor/fraidycat#94]
**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Firefox
 - Version: 74

**Describe the bug**
__Penny-Arcade latest update__:

FraidyCat: `Comic: Now You&#8217;ve Done It`
Penny-Arcade: `Now You’ve Done It`

The `’` failed to render correctly.

RSS Entry
```
              <item>
                <title><![CDATA[Comic: Now You&#8217;ve Done It]]></title>
                <link>https://www.penny-arcade.com/comic/2020/03/11</link>
                <description><![CDATA[New Comic: Now You&#8217;ve Done It]]></description>
                <author>Tycho@penny-arcade.com (Tycho)</author>
                <pubDate>Wed, 11 Mar 2020 07:01:00 PST</pubDate>
                <guid isPermaLink="false">/comic/2020/03/11/now</guid>
            </item>
```

**Comment by kickscondor** [comment]
Yes I'm seeing &amp;s in eli.li as well. Time to do some work on the HTML handling code.

Thanks, Jason!

**Comment by kickscondor** [comment]
Ok - a fix in is - should be out in Fraidycat 1.1.3 tonight.
